igor-web/src/main/groovy/com/netflix/spinnaker/igor/health/PollingMonitorHealth.groovy

```diff
@@ -46,8 +50,9 @@ public class PollingMonitorHealth implements HealthIndicator {
                 if (poller.lastPoll == null) {
                     healths << Health.unknown().withDetail("${poller.name}.status", 'not polling yet').build()
                 } else {
-                    // Check if twice the polling interval has elapsed.
```

Comment:
this comment almost matched what the code was doing

Reply:
hah